docs/getting_started/index.rst

```diff
@@ -1,17 +1,18 @@
 .. _getting-started:
 
 ===============
-Getting Started
+Developer Guide
 ===============
 
 Oras Python is a Python client and API for OCI registry as Storage.
 You should be able to easily pull and push artifacts from Python and build
-your own custom interactions.
+your own custom interactions. The main getting started docs can be found on
+`oras.land Python's page <https://oras.land/client_libraries/1_python/>`_ and
+the developer user guide (and more detailed examples) can be found here.
```

Comment:
developer user guide => developer guide?

Reply:
I meant to say "developer user" - a developer user is not a user of the client, but a developer developing with it. I guess maybe it sounds weird, but in my mind:

- developer: sort of referring to a main maintainer (will likely respond to issues / fix bugs)
- developer user: using _someone else's_ SDK, etc, (will likely open issues / report bugs)
- user: typically someone that is just, for example, running the client (but not developing anything)